src/web_app/html/index_template.html

```diff
@@ -124,24 +124,23 @@
 {% if not chromeapp %}
   <script type="text/javascript">
     var loadingParams = {
```

Comment:
Here I would expect a simple:
 var loadingParams = {{loadingParams}};

In python code i would expect like:
  loadginParams = get_room_parameters(self.request, None, None, None)
  self.write_response('index_template.html', {'loadingParams': json.dumps(loadingParams) })

Also whenever adding new params it would mean one less place to edit the code.